help/destinations/catalog/crm/salesforce.md

```diff
@@ -168,26 +173,40 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
     * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on these attributes.
     ![Platform UI screenshot showing Target mapping for LastName.](../../assets/catalog/crm/salesforce/target-mapping-lastname.png)
 
-    * For instance, depending on the values you want to update, add the following mapping between your XDM profile schema and your [!DNL Salesforce CRM] instance:
-    |XDM Profile Schema|[!DNL Salesforce CRM] Instance|
-    |---|---|
-    |`person.name.firstName`|`FirstName`|
-    |`person.name.lastName`|`LastName`|
-    |`personalEmail.address`|`Email`|
+    * If you are working with *Contacts* within your segment, refer to the Object Reference in Salesforce for [Contact](https://developer.salesforce.com/docs/atlas.en-us.object_reference.meta/object_reference/sforce_api_objects_contact.htm) and make a note of the mandatory fields.
```

Comment:
"make a note of the mandatory fields."

^^ which are the required fields? where can I see them so I make note of them? Also, "make note of something" sounds vague. What are we instructing users to do here? Remember, we need to be as precise as possible in the documentation and vague instructions will only leave the reader guessing.